spotlight-client/src/SystemNarrativePage/Section.tsx

```diff
@@ -77,10 +78,12 @@ const Section: React.FC<{ section: SystemNarrativeSection }> = ({
 }) => {
   return (
     <Container>
-      <SectionCopy>
-        <SectionTitle>{section.title}</SectionTitle>
-        <SectionBody>{HTMLReactParser(section.body)}</SectionBody>
-      </SectionCopy>
+      <Sticker>
```

Comment:
hm yeah ... it varies a bit by browser in my experience but it especially does not like it if you try to scroll while the animation is still in progress. It's definitely still kinda buggy! I think though that I want to collect some more feedback and deal with this animation tuning in a separate PR (since I don't think it's directly affected by any of the changes in this one)